src/program.rs

```diff
@@ -90,6 +90,8 @@ impl Drop for Program {
     }
 }
 
+unsafe impl Send for Program {}
```

Comment:
`Program`s are intermediate objects that are used to create `Kernel`s.  
I cannot see  a need to send them between threads, can you?

Reply:
I thought I need it (my current code does), but I revisited it and it turns out I don't actually need it. Thanks for making me having another look.